src/osd/OSD.cc

```diff
@@ -9220,7 +9222,7 @@ void OSD::handle_fast_pg_create(MOSDPGCreate2 *m)
 	    std::make_shared<PGPeeringEvent>(
 	      m->epoch,
 	      m->epoch,
-	      NullEvt(),
+	      PgCreateEvt(),
```

Comment:
Should the same be applied to `OSD::handle_pg_create()`? Or the issue is only reproducible with `OSD::handle_fast_pg_create`?

Reply:
Great work narrowing down the cause btw!

Reply:
> Should the same be applied to `OSD::handle_pg_create()` (according to your findings)? Or the issue is only reproducible with `OSD::handle_fast_pg_create`?

The same issue occurs in OSD::handle_pg_create() as well.